<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>451</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MK Updater</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userNametxt">
       <property name="placeholderText">
        <string extracomment="Username"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordtxt">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string extracomment="Password"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="unitClubBtn">
       <property name="text">
        <string>Unit Club</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inTouchBtn">
       <property name="text">
        <string>MK InTouch</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="centerOnScroll">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string extracomment="Test"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>inTouchBtn</zorder>
  <zorder>userNametxt</zorder>
  <zorder>passwordtxt</zorder>
  <zorder>unitClubBtn</zorder>
  <zorder>plainTextEdit</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>line</zorder>
  <zorder>unitClubBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
